recipes/nexusformat/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "nexusformat" %}
+{% set version = "0.4.20" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "58023fe0913a6d8273afc971cdc64ec0fc909f5d7d39a7819df0674ca2108ec2"
+
+build:
+  number: 0
+  entry_points:
+    - nxstack=nexusformat.scripts.nxstack:main
```

Comment:
I think there should also be a `nxduplicate` entrypoint?

Reply:
`nxduplicate` is only in master at this point, and not yet part of a release.